src/analysis/retail/shaman/elemental/modules/Abilities.tsx

```diff
@@ -8,8 +9,8 @@ class Abilities extends CoreAbilities {
     const combatant = this.selectedCombatant;
     return [
       {
-        spell: SPELLS.LAVA_BURST.id,
-        charges: combatant.hasTalent(SPELLS.ECHO_OF_THE_ELEMENTS_TALENT_ELEMENTAL.id) ? 2 : 1,
+        spell: SPELLS.LAVA_BURST_DAMAGE.id,
```

Comment:
```suggestion
        spell: TALENTS.LAVA_BURST_TALENT.id,
```

this should likely reference the talent. the `_DAMAGE` spells are typically not cast by players, but are used in `damage` events.